extensions/http/src/main/java/org/mule/extension/http/api/listener/HttpListenerProvider.java

```diff
@@ -106,6 +118,10 @@
     @Inject
     private MuleContext muleContext;
 
+    private ThreadingProfile workerThreadingProfile;
```

Comment:
shouldn't you then add the same TODO you added below and explain that depending on what we do this might be a parameter?